frame/example-offchain-worker/src/lib.rs

```diff
@@ -149,6 +161,22 @@ decl_storage! {
 }
 
 impl<T: Trait> Module<T> {
+	fn check_if_should_proceed(_block_number: T::BlockNumber) -> Result<(), &'static str> {
+		let s_next_at = StorageValueRef::persistent(b"example-offchain-worker::next-at"); // TODO: Rename after OCW renamed
+
+		if let Some(Some(next_at)) = s_next_at.get::<T::BlockNumber>() {
```

Comment:
This should use mutate instead of get and set. https://substrate.dev/rustdocs/v3.0.0/sp_runtime/offchain/storage/struct.StorageValueRef.html#method.mutate